subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoVersionIntegTest.groovy

```diff
@@ -43,7 +43,7 @@ class JacocoVersionIntegTest extends MultiVersionIntegrationSpec {
         then:
         def report = htmlReport()
         report.totalCoverage() == 100
-        report.jacocoVersion() == version
+        report.jacocoVersion() == version || report.jacocoVersion().startsWith(version)
```

Comment:
Can we move that detail into the report fixture? e.g., so we can do something like:

`report.assertJacocoVersion(version)`

And `assertJacocoVersion` does the fuzzy match.